app/src/main/java/com/appunite/loudius/network/utils/AuthFailureHandler.kt

```diff
@@ -21,11 +21,8 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.SharedFlow
 import kotlinx.coroutines.launch
-import javax.inject.Inject
-import javax.inject.Singleton
 
-@Singleton
-class AuthFailureHandler @Inject constructor(
+class AuthFailureHandler(
```

Comment:
**question**: I wonder if this still be singleton, cause is not yet annotated by `@Signelton` and it's not declared in a module as singleton.